src/terminal/parser/ut_parser/OutputEngineTest.cpp

```diff
@@ -284,6 +325,12 @@ class Microsoft::Console::VirtualTerminal::OutputEngineTest final
         VERIFY_ARE_EQUAL(mach._state, StateMachine::VTStates::CsiParam);
         mach.ProcessCharacter(L'J');
         VERIFY_ARE_EQUAL(mach._state, StateMachine::VTStates::Ground);
+
+        VERIFY_ARE_EQUAL(mach._parameters.size(), 4);
```

Comment:
I honestly do not understand why the actual parameter values are not even checked in this case. I mean that's sorta the whole point of this case, to get the correct parameters.